mem.go

```diff
@@ -2,12 +2,60 @@ package sqrl
 
 import (
 	"fmt"
+	"math/big"
+	"os"
 
+	"github.com/dustin/go-humanize"
+	"github.com/likexian/host-stat-go"
 	"github.com/shirou/gopsutil/mem"
 )
 
-func MemInfo() {
-	v, _ := mem.VirtualMemory()
+type Ram struct {
+	total, free, swapSpace string
+}
+
+type OS struct {
+	hostname, kernel, version string
+}
+
+func ramOSInfo() (Ram, OS) {
+	v, e := mem.VirtualMemory()
+	if e != nil {
+		fmt.Println(e)
+	}
+	s, err := mem.SwapMemory()
+	if err != nil {
+		fmt.Println(err)
+	}
+	i, er := host_stat.GetHostInfo()
+	if er != nil {
+		fmt.Println(er)
+	}
+	total := v.Total
+	free := v.Free
+	swap := s.Free
+	totalint := int64(total)
```

Comment:
Lets make these next three lines more simple.

``` go
totalint := int64(total)
totalbig := big.NewInt(totalint)
totalmb := humanize.BigBytes(totalbig)
```

can just become:

``` go
totalmb := humanize.BigBytes(big.NewInt(int64(totalint)))
```

Do that for the others ones as well, please.